src/core/tests/Test_StatusHandler.py

```diff
@@ -24,6 +25,7 @@
 
 class TestStatusHandler(unittest.TestCase):
     def setUp(self):
+        self.test_internal_file_size_limit = 126 * 1024
```

Comment:
Why is this size limit in the PR? This PR is creating *.complete.status file and copying it's contents to .status file. There is no logic on truncation in here. Please remove this and all it's related tests